src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -292,6 +292,9 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }, "collapseFrame");
         separatedPanel.slidingPanel.events.afterPanelShow.addListener(function () {
             separatedPanel.iframeRenderer.iframe.show();
+
+            // FLUID-6183: Required for bug in MS EDGE that clips off the bottom of adjusters
+            var height = separatedPanel.iframeRenderer.iframe.height();
```

Comment:
Assigning to a variable that is never uses causes a linting error. Does it have to be assigned to a variable to get the MS Edge fix? If so, you'll need to use an inline comment to disable the rule (see: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments ).